shotover-proxy/tests/test-configs/kafka/cluster-2-racks/docker-compose-kafka-3.9.yaml

```diff
@@ -9,7 +9,7 @@ networks:
           gateway: 172.16.1.1
 services:
   kafka0:
-    image: &image 'bitnami/kafka:3.9.0-debian-12-r3'
+    image: &image 'bitnami/kafka:3.9.0-debian-12-r6'
```

Comment:
This test case is now identical to `shotover-proxy/tests/test-configs/kafka/cluster-2-racks/docker-compose.yaml` and should be removed.